bundle/android/strings.xml

```diff
@@ -346,8 +346,6 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="gutenberg_native_ungroup" tools:ignore="UnusedResources">Ungroup</string>
     <string name="gutenberg_native_unsupported" tools:ignore="UnusedResources">Unsupported</string>
     <string name="gutenberg_native_updates_the_title" tools:ignore="UnusedResources">Updates the title.</string>
-    <string name="gutenberg_native_upgrade_your_plan_to_upload_audio" tools:ignore="UnusedResources">Upgrade your plan to upload audio</string>
-    <string name="gutenberg_native_upgrade_your_plan_to_use_video_covers" tools:ignore="UnusedResources">Upgrade your plan to use video covers</string>
```

Comment:
@geriux I'm curious why [these strings were added in `1.77.0`](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4888/commits/b4932dbdc1d6463c3671e9a0aa99ee9ca9e8be19) and then removed in this PR 🤔 . As far as I checked, they come from Jetpack ([reference](https://github.com/Automattic/jetpack/blob/bf63ca0d4ca0a700fbdd2056cf987fe4428c5705/projects/js-packages/shared-extension-utils/src/plan-utils.js#L137-L152)) but I think they're not actually used in the editor.

Reply:
Do they show up again in `1.77.1` ?

Reply:
> Do they show up again in 1.77.1 ?

Yep, after running the command `npm run bundle`, I see that these strings are re-introduced ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4923/files)).

I've checked this further and looks like the strings are expected to be here although they are not used. The file where they are defined ([reference](https://github.com/Automattic/jetpack/blob/master/projects/js-packages/shared-extension-utils/src/plan-utils.js)) it's being imported in a native file, hence as they're referenced, they're considered to be part of the editor and to be translated.

- These strings are defined in [the function `usableBlockWithFreePlan`](https://github.com/Automattic/jetpack/blob/bf63ca0d4ca0a700fbdd2056cf987fe4428c5705/projects/js-packages/shared-extension-utils/src/plan-utils.js#L137-L152).
- `usableBlockWithFreePlan` is used in the [exported function `isStillUsableWithFreePlan`](https://github.com/Automattic/jetpack/blob/bf63ca0d4ca0a700fbdd2056cf987fe4428c5705/projects/js-packages/shared-extension-utils/src/plan-utils.js#L172-L173).
- [`isStillUsableWithFreePlan` is exported](https://github.com/Automattic/jetpack/blob/bf63ca0d4ca0a700fbdd2056cf987fe4428c5705/projects/js-packages/shared-extension-utils/index.js#L13) as part of the [`@automattic/jetpack-shared-extension-utils` package](https://github.com/Automattic/jetpack/tree/master/projects/js-packages/shared-extension-utils).
- Package [`@automattic/jetpack-shared-extension-utils` package](https://github.com/Automattic/jetpack/tree/master/projects/js-packages/shared-extension-utils) is imported in [the native file `register-jetpack-block.native.native.js`.](https://github.com/Automattic/jetpack/blob/bf63ca0d4ca0a700fbdd2056cf987fe4428c5705/projects/plugins/jetpack/extensions/shared/register-jetpack-block.native.js#L1)

Reply:
Probably, we haven't bumped into this change before because [the Jetpack reference was recently updated](https://github.com/wordpress-mobile/gutenberg-mobile/commit/f70ae93007c7fefe410f0f70d4f1a099841b54a7), and a lot of time has passed since the last update.

Reply:
I see I did encounter issues with the jetpack reference, I'm sure it's related. Sorry for that 😅